.drone.yml

```diff
@@ -13,16 +13,14 @@ pipeline:
     image: lampepfl/dotty:2017-10-20
     commands:
       - cp -R . /tmp/1/ && cd /tmp/1/
-      - ./project/scripts/sbt testAll
-      - ./project/scripts/sbt ";dotty-bench/jmh:run 1 1 tests/run/arrays.scala"
+      - ./project/scripts/sbt ";compile ;testAll ;dotty-bench/jmh:run 1 1 tests/pos/alias.scala"
 
   test_bootstrapped:
     group: test
     image: lampepfl/dotty:2017-10-20
     commands:
       - cp -R . /tmp/2/ && cd /tmp/2/
-      - ./project/scripts/sbt dotty-bootstrapped/testAll
-      - ./project/scripts/sbt ";dotty-bench-bootstrapped/jmh:run 1 1 tests/run/arrays.scala"
+      - ./project/scripts/sbt ";dotty-bootstrapped/testAll ;dotty-bench-bootstrapped/jmh:run 1 1 tests/pos/alias.scala"
```

Comment:
Maybe add `;dotty-bootstrapped/compile` here too

Reply:
Won't `dotty-bootstrapped/testAll` do the compilation step already?

Reply:
@nicolasstucki You said at some point that `testAll` didn't build DottyDoc, right?

Reply:
It was not. Or at least it was missing some part of it.